<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0" colspan="5">
     <layout class="QHBoxLayout" name="MainButtonsLayout">
      <item>
       <widget class="QToolButton" name="ButtonSaveProfile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Save Profile</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::MenuButtonPopup</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonDeleteProfile">
        <property name="text">
         <string>Delete Profile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonLoadProfile">
        <property name="text">
         <string>Load Profile</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QTabWidget" name="MainTabBar">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <widget class="QWidget" name="TabInformation">
       <attribute name="title">
        <string>Information</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTabWidget" name="InformationTabBar">
          <property name="tabPosition">
           <enum>QTabWidget::West</enum>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabSettings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QToolBox" name="SettingsTabBar"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="4" column="0" colspan="5">
     <layout class="QHBoxLayout" name="DetectorLayout">
      <item>
       <widget class="QLabel" name="ConncetionLabel">
        <property name="text">
         <string>Connect to server</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="InputServerAddress">
        <property name="toolTip">
         <string>ip in dotted decimal, port is optional</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>127.0.0.1:1234</string>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>ip:port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="InputUsername">
        <property name="toolTip">
         <string>Username</string>
        </property>
        <property name="placeholderText">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="InputPassword">
        <property name="toolTip">
         <string>Password</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="ConnectionProgressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonStartConnection">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonEndConnection">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="ActionSaveProfile">
   <property name="text">
    <string>Save Profile</string>
   </property>
   <property name="toolTip">
    <string>Save Profile</string>
   </property>
  </action>
  <action name="ActionSaveProfileAs">
   <property name="text">
    <string>Save Profile As...</string>
   </property>
   <property name="toolTip">
    <string>Save Profile with custom name</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
